components/kyma-environment-broker/internal/process/upgrade_kyma/connectivity_bind.go

```diff
@@ -82,28 +83,29 @@ func (s *ConnectivityUpgradeBindStep) Run(operation internal.UpgradeKymaOperatio
 			operation.Connectivity.Instance.ProvisioningTriggered = false
 		}, log)
 		if retry > 0 {
+			log.Errorf("unable to update operation")
 			return operation, time.Second, nil
 		}
 		operation = op
 	} else {
 		// get the credentials from encrypted string in operation.Connectivity.Instance.
```

Comment:
Please check my 2 comments to `provisioning/connectivity_bind `step. They apply here as well.